ui-design/virtual-machines/create-vm/create-vm.md

```diff
@@ -0,0 +1,111 @@
+# Create VM
+
+Users can either import an existing virtual machine or create a new one within the Create VM flow.
+
+## Import existing VM
+
+### Launch Create Wizard from UI
+
+![Create virtual machine](img/1-1-virtual-machines.png)
+
+When no virtual machines exist, the Virtual Machines section of the UI will include quick access to the Create Virtual Machine wizard in its empty state.
+
+### Select Import
+
+![Import VM selection](img/1-2-1-import-choices.png)
+
+The wizard begins by asking the user whether they'd like to create or import a virtual machine.
+
+### Import VM Wizard - Basic Information
+
+![Import VM basic information](img/1-2-2-import-basic.png)
+
+The user first needs to provide the Source of the Import. Based on this selection, the fields below may change.
+
+![Import VM basic information filled](img/1-2-3-import-basic-filled.png)
+
+After providing a URL and credentials, the user can move on to the next step where a list of importable VMs will be displayed.
+
+### Import VM Wizard - Virtual Machines Selection
+
+![Import VM selection](img/1-2-4-import-virtual-machines.png)
+
+The user can select which VMs to import and optionally start them automatically after import. Clicking `Import` will kick off the process.
+
+### VM Import Notification
+
+![Import VM notification](img/1-2-5-import-notification.png)
+
+The user will be taken back to the Virtual Machines list view where they will see a toast notification along with the newly-imported VMs. Progress of each import will be shown in the list.
+
+## Create new VM
+
+### Select Create New Virtual Machine
+
+![Create VM selection](img/1-3-1-create-choices.png)
+
+Rather than importing, the user could choose to create a new virtual machine.
+
+### New VM Wizard - Basic Settings
+
+![Create VM basic settings](img/1-3-2-create-basic.png)
+
+The first step surfaces basic settings. Actions to proceed are only available after the user fills out all required fields.
+
+![Create VM basic settings filled](img/1-3-3-create-basic-iso.png)
+
+If the user selects `ISO` as the Provision Source, an additional ISO Path field will be displayed. The Operating System may be able to be pulled from the image selected, but in any case, the user can choose one from the dropdown.
+
```

Comment:
Selecting a file name doesn't make sense to me.  Where does that file come from?  Some of the designs we have been kicking around involve ISOs being "deployed" as services in the cluster where they are discoverable.  Another option would be to use a registryDisk.  The UI could then list these and allow the user to choose one.